File: CursoPOOUber/Java/Car.java
class Car {
    Integer id;         //Atributos   
    String license;     //Atributos
    Account driver;      //Atributos
    Integer passenger;  //Atributos

    //Método Constructor
    public Car(String license, Account driver){     //Asignamos los parametros obligatorios para
        this.license = license;                     //Para nuestro método constructor
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("License: " + license + "\nDriver: " + driver.name);  // Método para Imprimir
    }                                                                           //Todas las variables
           
}

File: CursoPOOUber/PHP/Car.php
<?php
require_once('account.php');
class Car {
    public $id;
    public $license;
    public $driver;
    public $passanger;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCAr(){
        echo "Licencia: $this->license Driver: ".$this->driver;
    }
}
?>